src/graphdb.js

```diff
@@ -52,6 +52,25 @@ function toPascalCase (str) {
     return r.trim();
 }
 
+// Changes every instance of invalid characters in the given label with the following abbreviations
+function replaceCleanseLabel(label) {
```

Comment:
The name of this function has two verbs and a noun. I would recommend shortening it to one verb and one noun.